lib/hyrax/transactions/work_create.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+require 'hyrax/transactions/transaction'
+
+module Hyrax
+  module Transactions
+    ##
+    # Creates a Work from a ChangeSet
+    #
+    # @since 3.0.0
+    class WorkCreate < Transaction
+      DEFAULT_STEPS = ['change_set.set_default_admin_set',
```

Comment:
I have to say, I really love the composition of these steps. They guide me towards an understanding of what goes into the "WorkCreate" transaction in a clearer means than the ActorStack